### ISSUE TITLE
A jvm crash occurs after FFmpegFrameGrabber is used in docker

### ISSUE THREAD

**Issue by mdzz0073** [bytedeco/javacv#1944]
When I used FFmpegFrameGrabber normally to get video file information, I was fine, but when I containerized the service, using FFmpegFrameGrabber caused a jvm crash

source code:
    public static String getVideoFps(File file){
        log.info("打开ffmpeg");
        FFmpegFrameGrabber grabber = new FFmpegFrameGrabber(file);
        String result = "";
        try {
            grabber.start();
            Integer frameRate =  (int) grabber.getVideoFrameRate();
            result = String.valueOf(frameRate);
            grabber.stop();
        }catch (Exception e){
        }
        return result;
    }

jvm crash when grabber.start();

this is error:
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x000000000000dc56, pid=14, tid=0x00007f8218d22b38
#
# JRE version: OpenJDK Runtime Environment (8.0_302-b08) (build 1.8.0_302-b08)
# Java VM: OpenJDK 64-Bit Server VM (25.302-b08 mixed mode linux-amd64 compressed oops)
# Derivative: IcedTea 3.20.0
# Distribution: Custom build (Wed Oct 20 22:15:10 UTC 2021)
# Problematic frame:
# C  0x000000000000dc56
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# java_error_9006.log
#
# If you would like to submit a bug report, please include
# instructions on how to reproduce the bug and visit:
#   https://icedtea.classpath.org/bugzilla

**Comment by saudet** [comment]
It looks like you're trying to use Alpine, in which case you'll need to use a version of OpenJDK that uses glibc.

**Comment by mdzz0073** [comment]
i try use openjdk1.8.0_345, but it didn't work。Whether I need to install glibc

**Comment by saudet** [comment]
To make this work, instead of using Alpine, you could also use CentOS or Ubuntu

**Comment by mdzz0073** [comment]
But I still had a jvm crash after installing glibc.

This is my error log:
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000000011466, pid=14, tid=0x00007f2e7097fb38
#
# JRE version: OpenJDK Runtime Environment (8.0_345-b01) (build 1.8.0_345-b01)
# Java VM: OpenJDK 64-Bit Server VM (25.345-b01 mixed mode linux-amd64 compressed oops)
# Derivative: IcedTea 3.24.0
# Distribution: Custom build (Tue Aug 23 09:53:14 UTC 2022)
# Problematic frame:
# C  0x0000000000011466
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid14.log
#
# If you would like to submit a bug report, please include
# instructions on how to reproduce the bug and visit:
#   https://icedtea.classpath.org/bugzilla

**Comment by saudet** [comment]
If you absolutely need to use that version of OpenJDK, you'll need to rebuild from source without glibc:
https://github.com/bytedeco/javacpp-presets/#build-instructions

**Comment by mdzz0073** [comment]
Can you tell me what version of opendjdk I should use

**Comment by saudet** [comment]
Any of the ones mentioned in the README.md file: https://github.com/bytedeco/javacv#required-software

**Comment by saudet** [comment]
Duplicate of https://github.com/bytedeco/javacpp/issues/329
